src/updates.js

```diff
@@ -241,6 +263,22 @@ class Updates {
   }
 }
 
+// Any logic to require a specific release when updating should go here
+const needsSpecificReleaseTag = (account, repository, platform, version) => {
+  const FIDDLE_TRANSITION_VERSION = "v0.35.1";
+
+  if (
+    [PLATFORM_ARCH.DARWIN_X64, PLATFORM_ARCH.DARWIN_ARM64].includes(platform) &&
+    account === "electron" &&
+    repository === "fiddle" &&
```

Comment:
Normally not worth the squeeze, but given the "requests per second" this service handles, you _might_ want to make the simple account check before checking for array inclusion. Then again, V8 might do that for us.

Reply:
Good point! I reordered them, and made the version check the second condition since that should lead to the fewest number of comparisons (assuming they're actually done in this order, as you said) by bailing out early once the majority of clients have updated past v0.35.1.